.github/CODEOWNERS

```diff
@@ -74,3 +74,8 @@
 /model/registry/container.yaml @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-container-approvers
 /model/registry/oci.yaml @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-container-approvers
 
+# .NET semantic conventions approvers
+/model/metrics/dotnet/ @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-dotnet-approvers  @open-telemetry/semconv-http-approvers
```

Comment:
The team is actually [semconv-dotnet-approver](https://github.com/orgs/open-telemetry/teams/semconv-dotnet-approver), can you update this?

Reply:
done, thanks!

Reply:
Not blocking - should we change the team name?

![image](https://github.com/open-telemetry/semantic-conventions/assets/17327289/77e2fd3a-830c-4539-9a73-f0754fcf937a)